internal/pkg/exec/docker.go

```diff
@@ -37,9 +43,10 @@ type BuildArguments struct {
 	URI        string            // Required. Location of ECR Repo. Used to generate image name in conjunction with tag.
 	Tags       []string          // Optional. List of tags to apply to the image besides "latest".
 	Dockerfile string            // Required. Dockerfile to pass to `docker build` via --file flag.
-	Context    string            // Optional. Build context directory to pass to `docker build`
-	Target     string            // Optional. The target build stage to pass to `docker build`
+	Context    string            // Optional. Build context directory to pass to `docker build`.
+	Target     string            // Optional. The target build stage to pass to `docker build`.
 	CacheFrom  []string          // Optional. Images to consider as cache sources to pass to `docker build`
+	Platform   *string           // Optional. OS/Arch to pass to `docker build`.
```

Comment:
I'm wondering if we should it as `string` here to keep it consistent with the other `string` fields such as `Target` 🤔?